libs/parallelism/algorithms/include/hpx/parallel/util/prefetching.hpp

```diff
@@ -31,7 +32,9 @@
 #endif
 
 namespace hpx { namespace parallel { namespace util {
-    namespace detail {
+
+    namespace prefetching {
```

Comment:
Did you have to change this because of name clashes?

Reply:
This was changed to simplify supporting the new CPOs. The types are not directly used by the user anyways.